app.js

```diff
@@ -2,9 +2,16 @@ const express = require('express')
 const app = express();
 const { psqlErrorHandler, customErrorHandler, serverErrorHandler } = require('./error-handlers')
 const { getTopics } = require('./controllers/topics.controllers')
+const endpoints = require('./endpoints.json')
 
 app.get('/api/topics', getTopics)
 
+app.use(express.json())
```

Comment:
This line is not needed yet and, as such, shouldn't be included in your code. We shouldn't have any code in our codebase that isn't being used yet.